makefiles/Makefile.win.mk

```diff
@@ -21,6 +21,10 @@ STATIC_PRE_LIB = $(OR_ROOT)lib\\
 STATIC_POST_LIB = .lib
 O=obj
 E=.exe
+L=.lib
```

Comment:
yes. I usually build and test csharp on unix first.
Thus my reservations about previous renaming.